packages/client/hmi-client/src/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -27,6 +27,7 @@ export interface OptimizeCiemssOperationState {
 	chartConfigs: string[][];
 	simulationsInProgress: string[];
 	simulationRunId: string;
+	optimzationRunId: string;
```

Comment:
Would recommend better naming; we are using `simulationId` rather loosely as anything-that-we-request-from-TA3, so if we are tracking multiple things it might be good to make things clear.

Reply:
good point, i think i will have `forecastRunId` and `optimzationRunId` and avoid the ambiguous simulation altogether